crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1512,10 +1515,7 @@ impl SymbolProvider for TempSymbolProvider {
         _frame: &mut dyn minidump_processor::FrameSymbolizer,
     ) -> Result<(), minidump_processor::FillSymbolError> {
         // TODO(ja): Deduplicate this. Probably should use a different map key, ...
```

Comment:
I have a sneaking suspicion that _maybe_ we can delete this now, but maybe it'd be good to confirm this with @jan-auer to be safe.